src/Symfony/Component/Validator/Mapping/ClassMetadata.php

```diff
@@ -262,6 +279,10 @@ public function addConstraint(Constraint $constraint)
      * @param Constraint $constraint The constraint
      *
      * @return ClassMetadata This object
+     *
+     * @throws GroupDefinitionException When the "class name" group has
```

Comment:
I think it makes sense to use `"class name"` group because the `"Default"` group can be explicitly defined without any problems ( please refer to the code example in my original post ), this issue only occurs when the `"class name"` group is explicitly defined.